pio-scripts/lto.py

```diff
@@ -0,0 +1,24 @@
+Import('env')
```

Comment:
Please add a short comment here why this script exists and what it does on a high level (I assume it replaces some parts of the linker with another that supports LTO?)